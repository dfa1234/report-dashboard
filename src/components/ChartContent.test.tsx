import { render } from "@testing-library/react";
import React from "react";
import { ChartContent } from "./ChartContent";

const sampleDatas = [
    ["Date", "Incomes", "Outcomes", "Revenues"],
    [new Date("2021-06-01"), 870, 59, 811],
    [new Date("2021-06-02"), 1133, 667, 466],
    [new Date("2021-06-03"), 621, 775, -154],
];

test("renders the chart", () => {
    const { baseElement } = render(<ChartContent data={sampleDatas} />);
    // the chart can't be tested with jest as explained here
    // https://github.com/rakannimer/react-google-charts/issues/214
    expect(baseElement).toMatchSnapshot();
});
