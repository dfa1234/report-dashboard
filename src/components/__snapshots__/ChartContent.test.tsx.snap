// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the chart 1`] = `
<body>
  <div />
</body>
`;
